.env

```
@@ -0,0 +1,2 @@
+VITE_API_URL=http://localhost:9000
+VITE_APP_PORT=9000
```

How about not to couple these names to a bundler? Today vite, tomorrow webpack.

```suggestion
API_URL=http://localhost:9000
APP_PORT=9000
```

---

`export default defineConfig(({ mode }) => {
    const env = loadEnv(mode, process.cwd(), '');
    return {
        plugins: [react()],
        server: {
            port: 5173,
        },
        test: {
            include: ['**/*.test.ts'],
        },
        define: {
            'process.env.API_URL': JSON.stringify(env.API_URL),

            // If you want to exposes all env variables, which is not recommended
            // 'process.env': env
        },
    };
});
`
the configuration file need to be changed then we can expose the custom env variable